resources/options_lookup.tsv

```diff
@@ -687,53 +687,59 @@ Water Heater Fuel	Fuel Oil
 		ResidentialAirflow																																													
-Water Heater Efficiency	"Electric Heat Pump, 50 gal"	ResidentialHotWaterHeaterHeatPump	storage_tank_volume=50.0	setpoint_temp=125.0	location=auto	element_capacity=4.5	min_temp=45.0	max_temp=120.0	cap=0.5	cop=2.8	shr=0.88	airflow_rate=181.0	fan_power=0.0462	parasitics=3.0	tank_ua=3.9	int_factor=1.0	temp_depress=0.0																														
+Water Heater Efficiency	"Electric Heat Pump, 50 gal"	ResidentialHotWaterHeaterHeatPump	storage_tank_volume=50.0	setpoint_temp=125.0	location=auto	element_capacity=4.5	min_temp=45.0	max_temp=120.0	cap=0.5	cop=2.8	shr=0.88	airflow_rate=181.0	fan_power=0.0462	parasitics=3.0	tank_ua=3.9	int_factor=1.0	temp_depress=0.0	setpoint_type=constant	schedules_directory=./resources	setpoint_schedule=hourly_setpoint_schedule.csv	operating_mode=standard	operating_mode_type=constant	operating_mode_schedule=hourly_operating_mode_schedule.csv																								
 		ResidentialAirflow																																													
-Water Heater Efficiency	"Electric Heat Pump, 50 gal, 140F"	ResidentialHotWaterHeaterHeatPump	storage_tank_volume=50.0	setpoint_temp=140.0	location=auto	element_capacity=4.5	min_temp=45.0	max_temp=120.0	cap=0.5	cop=2.8	shr=0.88	airflow_rate=181.0	fan_power=0.0462	parasitics=3.0	tank_ua=3.9	int_factor=1.0	temp_depress=0.0																														
+Water Heater Efficiency	"Electric Heat Pump, 50 gal, 140F"	ResidentialHotWaterHeaterHeatPump	storage_tank_volume=50.0	setpoint_temp=140.0	location=auto	element_capacity=4.5	min_temp=45.0	max_temp=120.0	cap=0.5	cop=2.8	shr=0.88	airflow_rate=181.0	fan_power=0.0462	parasitics=3.0	tank_ua=3.9	int_factor=1.0	temp_depress=0.0	setpoint_type=constant	schedules_directory=./resources	setpoint_schedule=hourly_setpoint_schedule.csv	operating_mode=standard	operating_mode_type=constant	operating_mode_schedule=hourly_operating_mode_schedule.csv																								
 		ResidentialAirflow																																													
-Water Heater Efficiency	"Electric Heat Pump, 80 gal"	ResidentialHotWaterHeaterHeatPump	storage_tank_volume=80.0	setpoint_temp=125.0	location=auto	element_capacity=4.5	min_temp=45.0	max_temp=120.0	cap=0.979	cop=2.4	shr=0.98	airflow_rate=480.0	fan_power=0.178	parasitics=8.5	tank_ua=4.0	int_factor=1.0	temp_depress=0.0																														
+Water Heater Efficiency	"Electric Heat Pump, 80 gal"	ResidentialHotWaterHeaterHeatPump	storage_tank_volume=80.0	setpoint_temp=125.0	location=auto	element_capacity=4.5	min_temp=45.0	max_temp=120.0	cap=0.979	cop=2.4	shr=0.98	airflow_rate=480.0	fan_power=0.178	parasitics=8.5	tank_ua=4.0	int_factor=1.0	temp_depress=0.0	setpoint_type=constant	schedules_directory=./resources	setpoint_schedule=hourly_setpoint_schedule.csv	operating_mode=standard	operating_mode_type=constant	operating_mode_schedule=hourly_operating_mode_schedule.csv																								

```

Comment:
Can we change "operating_mode_type" to "operating_mode_schedule_type" ? Seems a little ambiguous otherwise.

Reply:
Yes! Just changed the name in the latest commit (https://github.com/NREL/resstock/commit/3369a83c1a7c63c5b80ab4171a41a8788d58c052) to avoid any confusion here.

HPWH will always be a stratified tank, yes. I haven't had any luck modeling them with a mixed tank because the HP performance is a function of the temperature in the bottom half of the tank (adjacent to the condenser), not the entire tank, and the control logic is so finnicky it would be really hard to approximate with a mixed tank. I didn't want this merge to change any of the energy consumption results so I left all of the other water heaters as mixed tanks, at least for now.

We actually could look at switching to stratified tanks now, but there'll be some energy impacts associated with it. @nmerket and I added in the new stratified tank formulation into E+ version 9.2, so there should be pretty negligible run time impacts from making the switch now. There will however be impacts to energy consumption and thermal comfort (this branch also adds in unmet shower metrics) associated with making the switch. I was planning to try to get this merged in with mixed tanks so there's no energy impact and then start looking at the impact of making the switch to stratified tanks by default.

Reply:
Thanks @jmaguire1 . That plan sounds good. I do like the idea of having stratified tanks by default so users don't need to remember to do that when applying WH DF measures.